crates/core/src/db/models.rs

```diff
@@ -124,6 +124,22 @@ pub struct MetadataCreator<'a> {
     pub verified: bool,
 }
 
+/// A row in the `token_accounts` table
+/// helpful for tracking exchanges of tokens
+#[derive(Debug, Clone, Queryable, Insertable, AsChangeset)]
+pub struct TokenAccount<'a> {
+    /// The address of this account
+    pub address: Cow<'a, str>,
+    /// The mint address of the token
+    pub mint_address: Cow<'a, str>,
+    /// The owner token
+    pub owner_address: Cow<'a, str>,
+    /// The amount of the token, often 1
+    pub amount: i64,
```

Comment:
is `amount` immutable?

Reply:
I believe it is mutable.

Reply:
will we need to reindex if amount changes, or is that not something we care about? @dandlezzz